src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,16 +2,25 @@
 
 public class HanoiRhymer extends defaultCountingOutRhymer {
 
-int totalRejected = 0;
+    private int totalRejected = 0;
 
     public int reportRejected() {
         return totalRejected;
     }
 
-    public void countIn(int in) {
-    if (!callCheck() && in > peekaboo())
+    @Override
+    public void addInValuesToNumbers(int in) {
```

Comment:
Changing the API of rhymers is not the goal. The API has been already established.